### ISSUE TITLE
Password encryption causes invalid outline characters displayed

### ISSUE THREAD

**Issue by hackinteach** [PyFPDF/fpdf2#732]
Describe the bug

`pdf.set_encryption` causes outline text to be invalid. I tested without encryption set, outline text seems to be normal

**with encryption**
<img width="1225" alt="with encryption" src="https://user-images.githubusercontent.com/2115896/227776639-a28e3129-575c-4d69-8b41-667165d04a62.png">

**Without encryption**
<img width="1342" alt="without encryption" src="https://user-images.githubusercontent.com/2115896/227776645-1e7de806-ac13-4b82-a014-dc268eb0869e.png">

**Minimal code**

```python
from fpdf import FPDF

pdf = FPDF()
pdf.add_page()
pdf.set_font("helvetica", size=12)
pdf.start_section("test")
pdf.write(txt="hello")
pdf.start_section("test", level=1)
pdf.set_encryption("test", "test")
pdf.output("test_enc.pdf")
```

**Environment**
* OS (Windows, Mac OSX, Linux flavour...): OS X
* Python version: 3.8.12
* `fpdf2` version used: v2.6.1

**Comment by andersonhc** [comment]
Thank you for catching this bug @hackinteach

fpdf2 is not encrypting the outlines, so when the client decrypts the string, it turning into gibberish.
I'll open a pull request to fix this problem shortly.

**Comment by Lucas-C** [comment]
@allcontributors please add @hackinteach for bug

**Comment by Lucas-C** [comment]
A fix has been released in v2.7.0: https://github.com/PyFPDF/fpdf2/releases/tag/2.7.0
